alvr/common/src/inputs.rs

```diff
@@ -96,6 +98,22 @@ macro_rules! controller_inputs {
                             button_type: ButtonType::$ty,
                             device_id: *HAND_RIGHT_ID,
                         },
+                    ),
+                    (
+                        *[<LEFT_ $inputs _ID>],
+                        ButtonInfo {
+                            path: [<LEFT_ $inputs _PATH>],
+                            button_type: ButtonType::$ty,
+                            device_id: *HAND_TRACKER_LEFT_ID,
+                        },
+                    ),
+                    (
+                        *[<LEFT_ $inputs _ID>],
+                        ButtonInfo {
+                            path: [<LEFT_ $inputs _PATH>],
+                            button_type: ButtonType::$ty,
+                            device_id: *HAND_TRACKER_RIGHT_ID,
+                        },
```

Comment:
How exactly does this work? Two entries with the same key in a hashmap

Reply:
uh yeah, i think also don't set buttons for the hand tracker devices

Reply:
uh yeah, actually the hand tracker devices don't receive buttons anyway...

Reply:
yea then that should just go

Reply:
I'd say people trace up the call stack to understand code, so where the c++ code controller updates is called by rust code.